### ISSUE TITLE
No Dice Pricing Tool

### ISSUE THREAD

**Issue by BigWil** [hivelocity/bare-metal-provisioning-module-for-whmcs#1]
Well I've gone through it all twice.  Seems the pricing tool addon isn't working.  Seems to configure the product okay, grouping and server fine an all that.  When I go to get some prices configured for it though it's just a blank list of nothing.

Also does "to your cron job table" mean the system cron? Or is there a table I'm not aware of?

Looking at the API logs there is lots to see here.  Shoot me an email at wilhatfield at gee mail dot com and I can give you guys some access so you can get it debugged.

**Comment by bstetler** [comment]
I will send you an email. Add the cronjob to the user that apache is running as. In vanilla servers with just apache it could be www-data or apache. On a cpanel user it will be the cpanel user and will you will be able to add in the control panel.

**Comment by bstetler** [comment]
The problem was actually the product was set to free. the product has to be set to recurring with at least a price of 0.00 it will then show up in the pricing tool. The Documentation is not great and needs to be updated. We also plan to create many more features making it easier to use.

**Comment by BigWil** [comment]
Barry,

Thanks for looking into it.  Correct, that I did not see anything in there
regarding adjustment to the price so I figured the price was going to be
100% handled by the module.  I did try other possibilities with price, but
they may have been out of order or something. May have even been before I
installed the pricing module.

I will keep you informed should I run into any other larger humps that may
need covered in the docs better.

Thanks again,

Wil Hatfield
